Example/Backpack/Utils/FeatureStories/Groups/CardGroups.swift

```diff
@@ -61,6 +61,16 @@ struct CardGroupsProvider {
                 presentableDevidedCard("With divider without padding") { cardVC in
                     cardVC.divisionDirection = .vertical
                     cardVC.padded = false
+                },
+                presentableDevidedCard("With solid divider with padding") { cardVC in
+                    cardVC.divisionDirection = .vertical
+                    cardVC.padded = true
+                    cardVC.dividerStyle = .solid
+                },
+                presentableDevidedCard("With solid divider without padding") { cardVC in
+                    cardVC.divisionDirection = .vertical
+                    cardVC.padded = false
+                    cardVC.dividerStyle = .solid
```

Comment:
Could you please introduce Docs screenshots as well. 
This is done by:

- editing the Screenshots.swift file
- adding a new entry for the solid line (You can do this after the existing dotted example)
- running ./scripts/take-screenshots in the root of the project (This will take approx 30min)

Alternatively, you can 'limit' the screenshots taken by adusting BackpackSnapshotTestCase line 30, setting runOnly to `var runOnly: Set<String>? = ["Cards"]` 

In that case, make sure to not commit the 'removed' screenshots, as the script does a 'clear-all' before taking new snapshots.